newrelic.yml

```diff
@@ -144,6 +144,10 @@ common: &default_settings
   # (sometimes referred to as real user monitoring or RUM).
   # browser_monitoring.auto_instrument: true
 
+  # If true, enables injection of Content Security Policy Nonce in browser monitoring scripts
+  # https://content-security-policy.com/nonce/
+  # browser_monitoring.content_security_policy_nonce: true
+
```

Comment:
Good thinking! This file is auto-generated, so the content will be added on release. It'll use the description provided in `default_source.rb`, so there's no need to backport any changes made to the description to this file.

Reply:
ok, so we must remove this part?

Reply:
Confirmed as removed with [fc1d71f](https://github.com/newrelic/newrelic-ruby-agent/pull/2544/commits/fc1d71fb3206385cadebca443f0a6cecadf82c93)